connect_ext_test.go

```diff
@@ -475,7 +476,7 @@ func TestConcurrentStreams(t *testing.T) {
 	server := memhttptest.NewServer(t, mux)
 	var done, start sync.WaitGroup
 	start.Add(1)
-	for i := 0; i < 100; i++ {
+	for i := 0; i < runtime.NumCPU()*2; i++ {
```

Comment:
Should we use `runtime.GOMAXPROCS` instead? I believe that `NumCPU` doesn't take cgroup resource limits into account. GOMAXPROCS doesn't either (by default), but at least it's configurable.

Reply:
Switched to `GOMAXPROCS`, should return NumCPU but not an issue. Added the same for memhttp package as it has a concurrency test too.